src/ui/Base/RegisteredNamedMethods.ts

```diff
@@ -296,7 +296,7 @@ Initialization.registerNamedMethod(
  *
  * It is important to call this method **before** executing the query. Otherwise, no `SearchAsYouType` event will be
  * logged, and you will get a warning message in the console. See
- * [Sending Custom Analytics Events](https://developers.coveo.com/x/KoGfAQ).
+ * [Sending Custom Analytics Events](https://docs.coveo.com/en/365/).
```

Comment:
Link to https://docs.coveo.com/en/2726/#logging-your-own-search-events instead and update link name accordingly.